docs/src/stream.rst

```diff
@@ -139,6 +139,10 @@ API
     be made several times until there is no more data to read or
     :c:func:`uv_read_stop` is called.
 
+    .. versionchanged:: 1.36.1 :c:func:`uv_read_start()` now consistently
+      returns `UV_EALREADY` when called twice, and `UV_EINVAL` when the
+      stream is closing.
```

Comment:
It might be worth noting what was inconsistent before.

Reply:
Done 👍